shell/browser/ui/file_dialog_gtk.cc

```diff
@@ -325,10 +457,10 @@ void ShowOpenDialog(const DialogSettings& settings,
 
 bool ShowSaveDialogSync(const DialogSettings& settings, base::FilePath* path) {
   FileChooserDialog save_dialog(GTK_FILE_CHOOSER_ACTION_SAVE, settings);
-  save_dialog.SetupSaveProperties(settings.properties);
```

Comment:
Why was this removed but `SetupOpenProperties` is still present?

Reply:
This seems in advertant. I will add it back.